shell/platform/android/BUILD.gn

```diff
@@ -584,6 +584,10 @@ zip_bundle("android") {
     ]
     deps += [ ":flutter_shell_native" ]
   }
+
+  if (flutter_runtime_mode == "debug" && current_cpu == "arm") {
```

Comment:
does the current cpu matter?

Reply:
This target is relatively slow to build, and we only run these tests on android_debug_unopt (armv7).

Reply:
If you mean current v. target, this file refers to current elsewhere and it seems like current == target .. but current is not the "host" CPU

Reply:
I see. could we add a comment?

Reply:
Done